features/full_tests/batch_2/native_on_error.feature

```diff
@@ -1,34 +1,32 @@
 Feature: Native on error callbacks are invoked
 
-  Scenario: on_error returning false prevents C signal being reported
-    When I run "CXXSignalOnErrorFalseScenario" and relaunch the app
-    And I configure Bugsnag for "CXXSignalOnErrorFalseScenario"
-    Then I should receive no requests
+    Scenario: on_error returning false prevents C signal being reported
+        When I run "CXXSignalOnErrorFalseScenario" and relaunch the app
+        And I configure the app to run in the "non-crashy" state
+        And I configure Bugsnag for "CXXSignalOnErrorFalseScenario"
+        And I wait to receive 3 logs
+        Then the "debug" level log message equals "No startupcrash events to flush to Bugsnag."
+        And I discard the oldest log
+        Then the "debug" level log message equals "No regular events to flush to Bugsnag."
+        And I discard the oldest log
+        Then the "debug" level log message equals "No regular events to flush to Bugsnag."
```

Comment:
I'd collapse these into a single Cucumber "step of steps".

Reply:
Added a step of steps here, and updated the docker-compose to use v4.7.0 on the CLI